docs/omg-preface.md

```diff
@@ -0,0 +1,52 @@
+# Preface
+
+## OMG
+
+Founded in 1989, the Object Management Group, Inc. (OMG) is an open membership,
+not-for-profit computer industry standards consortium that produces and
+maintains computer industry specifications for interoperable, portable, and
+reusable enterprise applications in distributed, heterogeneous environments.
+Membership includes Information Technology vendors, end users, government
+agencies, and academia.
+
+OMG member companies write, adopt, and maintain its specifications following a
+mature, open process. OMG’s specifications implement the Model Driven
+Architecture® (MDA®), maximizing ROI through a full-lifecycle approach to
+enterprise integration that covers multiple operating systems, programming
+languages, middleware and networking infrastructures, and software development
+environments. OMG’s specifications include: UML® (Unified Modeling Language™);
+CORBA® (Common Object Request Broker Architecture); CWM™ (Common Warehouse
+Metamodel); and industry-specific standards for dozens of vertical markets.
+
+More information on the OMG is available at <https://www.omg.org/>.
```

Comment:
@zvr Should we also convert this <link> in OMG Preface to [link](link) as well?

Reply:
There are also few others.

Reply:
We shouldn't render the OMG Preface page on the html version of the spec - it is only to be incorporated in the version of the spec that we bring to OMG.

Reply:
Thank you. 

I'm asking because I was told last time that links shouldn't be in bare URLs inside a paragraph but instead there should be a link text, and an URL will be rendered as a footnote.

So instead of this paragraph:

`More information on the OMG is available at <https://www.omg.org/>.`

which will be rendered as:

> More information on the OMG is available at https://www.omg.org/.

We should have something similar to this paragraph:

`More information on the OMG is available at [the OMG website](https://www.omg.org/).`

Which will be rendered something like:
> More information on the OMG is available at the OMG website [2].

with `[2] https://www.omg.org/` as a footnote.

Please correct me if I misunderstand this. Thank you.

(Note: If there are differences in chapters, we may need two mkdocs.yml. One for PDF and one for HTML. Will figure this out.)